utils/smartpaste/htmlfix.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function fixNestedLists(doc: Document): Document {
```

Comment:
Could you also add a comment explaining what this is doing?

It looks like something isn't wrapping nested lists in `<li>` tags correctly. Is that something we're doing wrong elsewhere or is that coming from somewhere else? We don't need to fix it now, obviously, but I'm curious if that's something that our markdown code is rendering wrong.

Reply:
No, it comes from spreadsheets mainly. Turndown doesn't support that, because is not valid, but most browsers accept it, I can add a comment explaining the why

Reply:
It's fun having to support crappy, malformed HTML all the time, isn't it? :p